src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonApiMetricsConfig.java

```diff
@@ -26,6 +26,14 @@ public interface JsonApiMetricsConfig {
   @WithDefault("sort.type")
   String sortType();
 
+  @NotBlank
```

Comment:
Sorry, forgot to ask earlier: could you add Javadocs explaining that these are tag names to use for specific metrics (brief explanation)?